task/common/identifier.go

```diff
@@ -20,7 +20,7 @@ func (i Identifier) Long() string {
 		return match[0]
 	}
 
-	name := normalize(string(i), 50)
+	name := normalize(string(i), 30)
```

Comment:
why 30 or 50 what is this magical number?

Reply:
It's the value for `truncate` on the human readable part of the identifier. Set it to 30 because we want the total length of the identifier to be less than 50 characters.

Reply:
Would be better in a named var just to not forget